server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -1209,7 +1209,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
     }
 
     public void testClusterRecoversAfterExceptionDuringSerialization() {
-        try (Cluster cluster = new Cluster(randomIntBetween(1, 5))) {
+        try (Cluster cluster = new Cluster(randomIntBetween(2, 5))) {
```

Comment:
This is a consequence of not trying to serialise anything in a one-node cluster, which weakens the (best-effort) check that we can always serialise cluster states to some extent before accepting them. I think this is ok, we don't see bugs here very much and we have tools for excising broken bits of cluster state anyway. Worth some thought, however, it's easy enough to replace this at the cost of a bit of wasted work in a one-node cluster.